spec/paths/timelines@global.yaml

```diff
@@ -0,0 +1,33 @@
+get:
+  tags:
+    - Timelines
+  summary: Retrieve a list of events in Global timeline
+  description: |
+    The list is always sorted by occuredTime in descending order.
+  parameters:
+    - $ref: "#/parameters/collectionLimit"
+    - $ref: "#/parameters/collectionOffset"
+    - $ref: "#/parameters/collectionFilter"
+    - $ref: "#/parameters/collectionCriteria"
```

Comment:
We don't have support for Filter or Criteria. We only support `eventTypes[]` array